data/tlg0007/tlg081/tlg0007.tlg081.perseus-eng4.xml

```diff
@@ -0,0 +1,2182 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+	<teiHeader xml:lang="eng">
+		<fileDesc>
+			<titleStmt>
+				<title type="work">The apophthegms or remarkable sayings of kings and great
```

Comment:
please revise to `<title>The Apopthegms or Remarkable Sayings of Kings and Great Commanders</title>`